backend/core/validations/user_validation/DateValidator.js

```diff
@@ -0,0 +1,9 @@
+function validateDate(birthday) {
```

Comment:
Does the variable really need to be named birthday? You could validate any date in principle.